autoload/elixir/util.vim

```diff
@@ -2,25 +2,29 @@ let s:SKIP_SYNTAX = '\%(Comment\|String\)$'
 let s:BLOCK_SKIP = "synIDattr(synID(line('.'),col('.'),1),'name') =~? '".s:SKIP_SYNTAX."'"
 
 function! elixir#util#is_indentable_at(line, col)
+  if a:col == -1 " skip synID lookup for not found match
+	  return 1
+  end
   " TODO: Remove these 2 lines
   " I don't know why, but for the test on spec/indent/lists_spec.rb:24.
   " Vim is making some mess on parsing the syntax of 'end', it is being
   " recognized as 'elixirString' when should be recognized as 'elixirBlock'.
   call synID(a:line, a:col, 1)
-  " This forces vim to sync the syntax.
-  syntax sync fromstart
+  " This forces vim to sync the syntax. Using fromstart is very slow on files
+  " over 1k lines
+  syntax sync minlines=20 maxlines=150
 
   return synIDattr(synID(a:line, a:col, 1), "name")
         \ !~ s:SKIP_SYNTAX
 endfunction
 
-function! elixir#util#is_indentable_match(line, pattern)
-  return elixir#util#is_indentable_at(a:line, match(getline(a:line), a:pattern))
+function! elixir#util#is_indentable_match(line, line_text, pattern)
```

Comment:
Since, now, we have the `line` dictionary, with `:num, :text` keys, I think we could use that dictionary here, instead of add one argument, wdyt?